src/qat/purr/backends/qblox/device.py

```diff
@@ -223,7 +223,7 @@ def install(self, package: QbloxPackage):
 
         sequence = asdict(package.sequence)
         if self.dump_sequence:
-            filename = f"schedules/sequence_{module.slot_idx}_{sequencer.seq_idx}_@_{datetime.utcnow()}.json"
+            filename = f"schedules/sequence_{module.slot_idx}_{sequencer.seq_idx}_@_{datetime.utcnow().strftime('%m-%d-%Y_%H%M%S')}.json"
```

Comment:
Do we need to worry about backwards compatibility here?

Reply:
nah this is only useful for debugging, also no scheme had been in place for post processing of the dumped files, so all good